internal/provider/resource_flink_artifact.go

```diff
@@ -73,22 +74,28 @@ func artifactResource() *schema.Resource {
 					}
 					return
 				},
-				Description: "The artifact file for Flink Artifact.",
+				Description: "The artifact file for Flink Artifact in JAR or ZIP format.",
 			},
 			paramRuntimeLanguage: {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice(acceptedRuntimeLanguage, true),
 				Default:      "JAVA",
-				Description:  "Runtime language of the Flink Artifact. The default runtime language is Java.",
+				Description:  "Runtime language of the Flink Artifact as Python or Java. The default runtime language is Java.",
```

Comment:
nit:
```suggestion
				Description:  "Runtime language of the Flink Artifact as Python or Java. The default runtime language is JAVA.",
```
"java" or "JAVA", might need some consistency with data source